src/routes/docs/tutorials/nuxt/step-7/+page.markdoc

```diff
@@ -0,0 +1,197 @@
+---
+layout: tutorial
+title: Ideas page
+description: Add ideas from Appwrite database in your app.
+step: 7
+---
+
+With the methods in the `useIdeas`composable we can get some ideas to the home page for the users to interact with.
+We will use it in a form component so the logged in users can add their ideas, and in a list component to render the ten most recent ideas.
+We start with building the form.
+
+# Idea form {% #idea-form %}
+
+On the home page, the logged in users should be able to add their ideas to the Appwrite database.
+The form need a text field for filling in the title, a textarea for the description and a submit button.
+
+From the `components` directory, add the file `ÌdeasForm.vue` and add the following code.
+
+```vue
+<!-- components/ideasForm.vue -->
+
+<template>
+  <div>
+    <article class="container padding-0">
+      <h4 class="heading-level-4">Submit Idea</h4>
+      <form @submit.prevent="handleAddIdea" class="u-margin-block-start-16">
+        <ul class="form-list">
+          <!-- Title input field -->
+          <li class="form-item">
+            <label class="label">Title</label>
+            <input
+              type="text"
+              placeholder="Title"
+              name="title"
+            />
+          </li>
+          <!-- Description input field -->
+          <li class="form-item">
+            <label class="label">Description</label>
+            <textarea
+              placeholder="Description"
+              name="description"
+            />
+          </li>
+          <button class="button" aria-label="Submit idea" type="submit">
+            Submit
+          </button>
+        </ul>
+      </form>
+    </article>
+  </div>
+</template>
+
+<script setup>
+const ideas = useIdeas();
+const user = useUserSession();
+
+const handleAddIdea = async (event) => {
+  const form = event.target;
+  const formData = new FormData(form);
+
+  // Extract the values from the FormData object and add userId
+  const postIdeaData = {
+    userId: user.current.value.userId,
+    title: formData.get('title'),
+    description: formData.get('description'),
+  };
+
+  await ideas.add(postIdeaData);
+
+  form.reset(); // Clear the form
+};
+</script>
+```
+
+Next, add the component to the page `pages/index.vue` by auto-importing it in the `<template>`tag.
+In doing that, we need to take a moment to think about how we want to display the form to the users.
+
+Since it should only be shown to logged in user, we need to wrap it in a `<section>`that renders conditionally when the `isLoggedIn` reference in the `useUserSession` is true.
```

Comment:
```suggestion
Since it should only be shown to logged in user, we need to wrap it in a `<section>` that renders conditionally when the `isLoggedIn` reference in the `useUserSession` is true.
```